backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/clamp.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+import numpy as np
+
+from nodes.properties.inputs import ImageInput, SliderInput
+from nodes.properties.outputs import ImageOutput
+
+from .. import adjustments_group
+
+
+@adjustments_group.register(
+    schema_id="chainner:image:clamp",
+    name="Clamp",
+    description="Clamps the values of an image.",
+    icon="ImContrast",
+    inputs=[
+        ImageInput(),
+        SliderInput(
+            "Minimum",
+            minimum=0.0,
+            maximum=1.0,
+            default=0.0,
+            precision=4,
+            controls_step=0.001,
+            scale="log",
```

Comment:
Does `log` make sense here? Wouldn't a linear scale better?

Reply:
I generally like the log scale for my sliders as it feels more exact at the lower end.